controllers/database_reconciler_helpers.go

```diff
@@ -213,25 +216,26 @@ func (r *DatabaseReconciler) handleSync(ctx context.Context, database *ndbv1alph
 		}
 
 		reqData := map[string]interface{}{
-			ndbv1alpha1.NDB_PARAM_PASSWORD:       dbPassword,
-			ndbv1alpha1.NDB_PARAM_SSH_PUBLIC_KEY: sshPublicKey,
+			common.NDB_PARAM_PASSWORD:       dbPassword,
+			common.NDB_PARAM_SSH_PUBLIC_KEY: sshPublicKey,
 		}
-
-		generatedReq, err := ndbv1alpha1.GenerateProvisioningRequest(ctx, ndbClient, database.Spec, reqData)
+		d := &wrapper_implementations.Database{Database: *database}
+		generatedReq, err := ndb_api.GenerateProvisioningRequest(ctx, ndbClient, d, reqData)
+		// generatedReq, err := ndb_api.GenerateProvisioningRequestt(ctx, ndbClient, database.Spec, reqData)
 		if err != nil {
 			log.Error(err, "Could not generate provisioning request, requeuing.")
 			return r.requeueOnErr(err)
 		}
 
-		taskResponse, err := ndbv1alpha1.ProvisionDatabase(ctx, ndbClient, generatedReq)
+		taskResponse, err := ndb_api.ProvisionDatabase(ctx, ndbClient, generatedReq)
 		if err != nil {
 			log.Error(err, "An error occurred while trying to provision the database")
 			return r.requeueOnErr(err)
 		}
 		// log.Info(fmt.Sprintf("Provisioning response from NDB: %+v", taskResponse))
 
 		log.Info("Setting database CR status to provisioning and id as " + taskResponse.EntityId)
-		database.Status.Status = ndbv1alpha1.DATABASE_CR_STATUS_PROVISIONING
+		database.Status.Status = common.DATABASE_CR_STATUS_PROVISIONING
```

Comment:
Can you please apply PascalCase here? @yash-ntnx please share your opinion on this one too.

NDB UI does the same - they change the case from uppercase to PascalCase. Also this seems to be the K8S standard so I'd highly recommend following that.

Reply:
Right, that would make sense!

Reply:
Addressed, made changes in the constants file.